src/ScottPlot/Plottable/RadialGaugePlot.cs

```diff
@@ -0,0 +1,709 @@
+
+        /// <summary>
+        /// Determines the gauge label position as a percentage of the gauge length
+        /// 0 being the beginning and 100 (default value) the ending of the gauge.
+        /// </summary>        
+        public float GaugeLabelPos
+        {
+            get => _GaugeLabelPos;
+            set => _GaugeLabelPos = value > 100 ? 100 : (value < 0 ? 0 : value);
+        }
+        private float _GaugeLabelPos = 100;
+
+        /// <summary>
+        /// Determines whether the gauges are drawn starting from the inside (default value) or from the outside.
+        /// </summary>
+        public RadialGaugeOrder GaugeOrder { get; set; } = RadialGaugeOrder.InsideToOutside;
+
+        /// <summary>
+        /// <see langword="True"/> if the gauges' background is adjusted to <see cref="StartingAngleGauges"/>.
+        /// Default value is set to <see langword="False"/>.
+        /// </summary>
+        public bool NormBackGauge { get; set; } = false;
+
+        /// <summary>
+        /// Angle (in degrees) at which the gauges start: 270° for North (default value), 0° for East, 90° for South, 180° for West, and so on.
+        /// Expected values in the range [0°-360°], otherwise unexpected side-effects might happen.
+        /// </summary>
+        public float StartingAngleGauges
+        {
+            get => _StartingAngleGauges;
+            set
+            {
+                _StartingAngleGauges = (float)ReduceAngle(value);
+                ComputeAngularData();
+            }
+        }
+        private float _StartingAngleGauges = 270f;
+
+        /// <summary>
+        /// The initial angle (in degrees) where the background gauges begin. Default value is 270° the same as <see cref="StartingAngleGauges"/>.
+        /// </summary>
+        public float StartingAngleBackGauges
+        {
+            get => _StartingAngleBackGauges;
+        }
+        private float _StartingAngleBackGauges = 270f;
+
+        /// <summary>
+        /// The empty space between gauges as a percentage of the gauge width.
+        /// Values in the range [0-100], default value is 50 [percent]. Other values might produce unexpected side-effects.
+        /// </summary>
+        public float GaugeSpacePercentage
+        {
+            get => _GaugeSpacePercentage;
+            set => _GaugeSpacePercentage = (value > 100 ? 100 : (value < 0 ? 0 : value));
+        }
+        private float _GaugeSpacePercentage = 50f;
+
+        /// <summary>
+        /// <see langword="Color"/> of the value labels drawn inside the gauges.
+        /// </summary>
+        public Color GaugeLabelsColor { get; set; } = Color.White;
+
+        /// <summary>
+        /// Size of the gague label text as a percentage of the gauge width.
+        /// Values in the range [0-100], default value is 75 [percent]. Other values might produce unexpected side-effects.
+        /// </summary>
+        public float GaugeLabelsFontPct
```

Comment:
I would rename this, maybe something like `GaugeLabelFontSize`?

I don't think the current name makes it clear that this changes font size, and in C# we often avoid abbreviations like `pct`. I also don't know if the fact that this is a fraction is as important as what it does (changes font size).

Reply:
I'll change this to `FontSizeFraction`